tests/post_training/pipelines/lm_weight_compression.py

```diff
@@ -275,7 +275,7 @@ def _validate(self):
                 load_in_8bit=False,
                 compile=False,
                 stateful=is_stateful,
-                ov_config={"DYNAMIC_QUANTIZATION_GROUP_SIZE": "0"},
+                ov_config={"DYNAMIC_QUANTIZATION_GROUP_SIZE": "32"},
```

Comment:
```suggestion
```
I suggest to use default parameters and update metrics taking into account it.

Reply:
makes sense, corrected